src/common/execute.go

```diff
@@ -58,11 +58,17 @@ func RunCommand(cmd *exec.Cmd, dir string, showOutput bool) error {
 	var wg sync.WaitGroup
 	c := make(chan int, 1)
 
+	var stdoutBuf bytes.Buffer
+	var stderrBuf bytes.Buffer
+
 	cmd.Dir = dir
 	if showOutput {
 		cmd.Stdout = os.Stdout
 		cmd.Stderr = os.Stderr
 	} else {
+		cmd.Stdout = &stdoutBuf
```

Comment:
Do you think that it's a good idea? Write a text (in any amount) to buffer? Output could be huge.

Reply:
Decided to store only last N bytes in such buffer.

I suggest doing in in a separated patch, added to checklist of #242